src/Api/User.php

```diff
@@ -47,10 +47,20 @@ public function __construct(Client $client, $onlineIdOrProfileObject = '')
         }
     }
 
+    /**
+     * Gets accountId ID.
+     *
+     * @return string
+     */
+    public function accountId() : string
+    {
+        return $this->info()->accountId;
```

Comment:
I've just approved another PR with this in, although this has a MUCH better name. Will submit my own PR to switch it to this name.